traffic_ops/app/lib/UI/ConfigFiles.pm

```diff
@@ -160,6 +160,37 @@ sub server_data {
 	return $server;
 }
 
+sub get_ats_major_version {
+	my $self 	 = shift;
+	my $server   = shift;
+
+	my $ats_ver =
+		$self->db->resultset('ProfileParameter')
+		->search( { 'parameter.name' => 'trafficserver', 'parameter.config_file' => 'package', 'profile.id' => $server->profile->id },
+		{ prefetch => [ 'profile', 'parameter' ] } )->get_column('parameter.value')->single();
+		
+	if (!defined $ats_ver) {
+	        $ats_ver = "5";
+            $self->app->log->error("Parameter package.trafficserver missing for profile ".$server->profile->name . ". Assuming version $ats_ver");
+        }
+
+	my $ats_major_version = substr( $ats_ver, 0, 1 );
```

Comment:
Is this going to fail disastrously at version 10? We're at 8 currently.

Reply:
Possibly, but we use this check elsewhere. So when we get to that point its going to fail in a lot of places, not just here